roles/cloud-bootstrap/tasks/network.yml

```diff
@@ -0,0 +1,66 @@
+---
+- name: Create control-plane network
+  os_network:
+    cloud: "{{ cloud }}"
```

Comment:
Do we have a default definition for `cloud`? I see `os_cloud` in the `create_hosts` role that this replaces, and a `cloud` attribute of `nodepool_providers`.